.github/workflows/assets.yml

```diff
@@ -79,7 +79,7 @@ jobs:
           # hand-off part between the two. Therefore for now unpack the kernel.
           if [ "${{ env.ARCH }}" = arm64 ]; then
              mv assets/kernel assets/kernel.gz
```

Comment:
The kernel is not compressed, I think this entire run can be removed...

Reply:
Then the whole code block including the FIXME comment can be dropped. Maybe leaving a comment saying that we assume the kernel is not compressed?

Reply:
well we don't decompress amd64 kernel, so I think it's safe just to drop this step without any comment. Now it's default behaviour